src/Kovan-DssSpell.t.sol

```diff
@@ -594,70 +596,185 @@ contract DssSpellTest is DSTest, DSMath {
         }
     }
 
-    function testSpellIsCast_GUSD_INTEGRATION() public {
+    function testRootExecuteSpell() public {
+        vote();
+        scheduleWaitAndCast();
+        assertTrue(spell.done());
+
+        DSTokenAbstract(oldChief.IOU()).approve(address(oldChief), uint256(-1));
+        oldChief.free(999999999999 ether);
+        gov.approve(address(newChief), uint256(-1));
+
+        newChief.lock(80_000 ether);
```

Comment:
I think its worth putting this repeated logic in a function.

Reply:
I almost did that, let me see how it looks.

Reply:
I'm finally leaving it as it is, the grouping itself is a bit weird as the functions do different kind of stuff.
I've also added another function that does a similar thing but not exactly the same, so it would still leave that one out of a grouping function.